openshift/distro/ubi/9/Dockerfile

```diff
@@ -54,7 +51,7 @@ ARG BUNDLE_WITHOUT=development:test
 COPY --chown=1001:1001 ${BUNDLE_GEMFILE}.lock "${HOME}/app/"
 
 RUN mkdir -p "${HOME}/.gem/bin" \
- && echo "gem: --bindir ~/.gem/bin" > "${HOME}/.gemrc"
+ && echo "gem: --bindir ~/.gem/bin --install-dir /usr/share/gems" > "${HOME}/.gemrc"
```

Comment:
This was weird. `gem` is configured by default to use `/usr/local/share/gems` as install dir, but somehow browsing through the fs I found the gem files are installed under `/usr/share/gems`. Not sure why this happens but manually providing the proper install dir is now required for `gem uninstall` below to work properly